tests/dat/manifest_data_compose_actions_for_docker.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements.  See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.  The ASF licenses this file to you
+# under the Apache License, Version 2.0 (the # "License"); you may not use this
+# file except in compliance with the License.  You may obtain a copy of the License
+# at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+packages:
+  IntegrationTestDocker:
+      actions:
+          OpenWhiskSkeleton:
+              docker: openwhisk/skeleton
+          OpenWhiskSkeletonWithNative:
+              native: true
+          CustomDockerAction1:
+              function: ../src/integration/docker/actions/exec.zip
+              docker: openwhisk/skeleton
+          CustomDockerAction2:
+              function: ../src/integration/docker/actions/exec.zip
+              native: true
+          CustomDockerAction3:
+              function: ../src/integration/docker/actions/bash
+              native: true
+          CustomDockerAction4:
+              function: ../src/integration/docker/actions/go
+              native: true
+          CustomDockerAction5:
+              function: ../src/integration/docker/actions/go
+              docker: mydockerhub/myimage
```

Comment:
great 👍 

should we assume that custom/private docker registries are supported as well, provided the machine has the credentials needed to connect to that registry ?

Reply:
@ddragosd yes and no, custom docker registries are supported but not private. OpenWhisk does not support private registries yet (see https://github.com/apache/incubator-openwhisk/issues/1362) since the docker image is pulled when you invoke that action for the first time so even if the machine from where the action was created has credentials, its not useful.